### PR TITLE
Added FAO parser

### PR DESCRIPTION
**Pull request by SyeedHasan**
I've managed to add the FAO dataset (not complete), and added the parser file.

The file, however, doesn't work since there's an issue with the getHeaders' function implementation. Should I add the complete dataset now?

### CODE REVIEW

File: src/csvToJson.js [hunk 1]
```diff
@@ -24,7 +25,7 @@ const fileWriter = (i, fileName, start, stop) => {
   // @TODO change that. it will work only for one case.
   // we can also create a method for path.join, so it wouldn't complicate our code
   // really bad line
-  const jsonPath = `/projects/USFA/${folderName}/${fileName}${i}.json`;
+  const jsonPath = `/projects/FAO/${folderName}/${fileName}${i}.json`;
```
Comment (atherdon) [inline src/csvToJson.js]: you should update this line.
As you can see - it was working with USFA before. When you change foldername - you destroying previous working script.

File: src/fileSystem.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
+// const { parseDirectoryFiles } = require('./index');
 // Here I want to put all methods, related to reading data, etc.
-import { readdir, existsSync } from "fs";
-import csvToJson from "./csvToJson";
+const { readdir, existsSync } = require("fs");
```
Comment (atherdon) [inline src/fileSystem.js]: wrong change. this whole module is converted to work with ES6 style.

**Comment by atherdon** [comment]
you've made a lot of wrong changes actually. it was a bad idea to convert code from ES6 to ES5.
Not sure what we need to do here: 1) close this PR so you'll work on it from start or you will roll back some of your changes?

**Comment by SyeedHasan** [comment]
Let me roll my changes. Is babel pre-configured to work with ES6? I was always receiving import errors.

**Comment by atherdon** [comment]
babel working fine

**Comment by atherdon** [comment]
will make a video later, explaining how build is working

**Comment by SyeedHasan** [comment]
Thanks. I'll roll my changes and try to fix other bugs.

**Comment by atherdon** [comment]
just to be sure - don't try to do all things at one pr. it better to have
more PRs with small changes, rather than just something.
so roll back stuff, related to ES6 and then buzz me at your PR.
Next task you'll do with another pull request

>

**Comment by SyeedHasan** [comment]
Got it. I'll pull a new request as soon as I've handled a feature or bug.

**Comment by atherdon** [comment]
but let's fix current issue first

>

**Comment by atherdon** [comment]
just a quick note - dont delete next time a PR template. it's better to fill it with details - so anyone can quickly understand what your feature is about

**Pull request closed without merging**